tools/enforce-tags/main.go

```diff
@@ -0,0 +1,38 @@
+package main
+
+import (
+	"fmt"
+	"log"
+	"os"
+	"os/exec"
+	"strings"
+
+	"github.com/sourcegraph/sourcegraph/enterprise/dev/ci/images"
+)
+
+func main() {
+	if len(os.Args) != 3 {
+		log.Fatal("constraint, directory arguments must be provided")
+	}
+	var (
+		constraint = os.Args[1]
+		dir        = os.Args[2]
+	)
+	args := []string{
+		"run",
+		"github.com/slimsag/update-docker-tags",
```

Comment:
we should probably make this a library and call it directly

Reply:
that would be nice in the long run - i think a first step would be a better path towards owning this tool ourselves, however: https://github.com/sourcegraph/sourcegraph/issues/16280